tests/checker/test_noproxy.py

```diff
@@ -16,11 +16,9 @@
 """
 Test proxy handling.
 """
+from test.support import EnvironmentVarGuard
+
 from . import httpserver
-try:
-    from test.support import EnvironmentVarGuard
-except ImportError:  # Python 3
-    from test.test_support import EnvironmentVarGuard
```

Comment:
Hey the comment claims EnvironmentVarGuard lived in test.test_support on Python 3!  The comment was wrong.

BTW regarding [test.support](https://docs.python.org/3/library/test.html#module-test.support):

> Note test.support is not a public module. It is documented here to help Python developers write tests. The API of this module is subject to change without backwards compatibility concerns between releases.

and we're naughty naughty people for using it.

Reply:
Yes, fortunately  it is only in the test suite and hopefully it will fail noisily if it changes